test/built-ins/AsyncGeneratorPrototype/return/return-state-completed-broken-promise.js

```diff
@@ -53,7 +53,7 @@ it.return(brokenPromise)
       throw new Test262Error("Expected rejection");
     },
     err => {
-      assert(unblocked, false, 'return should be rejected before generator is resumed');
+      assert(unblocked, 'return should be rejected when the generator is completed');
```

Comment:
```suggestion
      assert(unblocked, 'return should be rejected after the generator is resume');
```

https://github.com/tc39/ecma262/issues/3596#issuecomment-2875186161

Reply:
@legendecas In this test, the state of the generator when `.return` is called is `~executing~`, not `~completed~`, because at the time `.return` is called the generator is paused at `await blocking`. So the quoted step doesn't apply.

Reply:
@bakkot thanks, yeah, it should be awaited. Updated the test to reflect the expected state should be `~completed~`.